index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<link href="https://fonts.googleapis.com/css?family=Indie+Flower|Laila:400,700" rel="stylesheet">
+		<title>Adventures in Programming Apprenticeship</title>
+		<link rel="stylesheet" href="css/style.css">
+	</head>
+	<body>
+		<header>
+			<h1>Dwayne Pate</h1>
+			<h2>Adventures in Design and Programming</h2>
+			<img src="images/dwayne_48.jpg" alt="Dwayne">
+			</header>
+		<article class="bio">
+			<h3>About Me</h3> 
+				After years in the education field, I decided to take action on my interest in computer programming.  I'm a student at Nashville Software School, where I'm steadly making progress on my goal!
+
+		</article>
+		<article class="links">
+			<h3>A collection of resources that made this journey easier!</h3>
+			<ul class="resource-links">
+				<li><a href="https://teamtreehouse.com/home">Team Treehouse</a>- Video based learning with coding practice and quizzed.  Earn badges and points as you learn!</li>
+				<li><a href="https://developer.mozilla.org/en-US/">Mozilla Developer Network</a>- If I have questions about HTML or JavaScript, this is my first stop.</li>
+				<li><a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">A Complete Guide to Flexbox</a>- Moving elements around your page with CSS has never been easier.</li>
+			</ul>
+
+
+		</article>
+		<article class="projects">
+			<section>
+				<h3>Future Project #1</h3>
+			
+			</section>
+
+			<section>
+				<h3>Future Project #2</h3>
+
+			</section>
+		</article>
+		<footer>
+			<ul>
+				<li><h3>Contact Information</h3></li>
+			</ul><br>
+			<ul>
```

Comment:
can we transform this area to look a little more interesting?  instead of links for these how about icons?  ie mail icon for email that is clickable.  github logo for github, twitter logo for twitter etc.